### ISSUE TITLE
Add `--out` flag to `plz build`

### ISSUE THREAD

**Issue by Tatskaari** [thought-machine/please#2927]
It would be useful to have a `--out` flag e.g. `plz build --out ./dir //src:please` to copy `plz-out/bin/src/please` to `./dir/please`. 

When set, the [build](https://github.com/thought-machine/please/blob/master/src/please.go#L457) subcommand would loop through all the original targets i.e. `state.ExpandOriginalLabels()`. copying their outputs to the given directory:

```golang
for _, target := range state.ExpandOriginalLabels() {
	outs := state.Graph.TargetOrDie(target).FullOutputs()
        // copy the outs
}
```

**Comment by peterebden** [comment]
I might argue for the name being `--out_dir` to be clearer about what it'll do. Like the idea otherwise though!

**Comment by DefinitelyNotBen** [comment]
Hey, happy to work on this if you can assign it to me

**Comment by Tatskaari** [comment]
> Hey, happy to work on this if you can assign it to me

I'm afraid I can't assign it to you if you're not part of the org, but please do have a crack!

**Comment by samwestmoreland** [comment]
Closed by #2950
